src/modules/app/tasks/add-task-form/add-task-form.component.html

```diff
@@ -11,6 +11,20 @@ <h1 class='title'>{{ 'titles.tasks-page' | translate }}</h1>
     <input matInput type="text"  placeholder="{{ 'placeholders.description' | translate }}" formControlName="taskDescription">
     <mat-error *ngIf="addTaskForm.get('taskDescription').hasError('required')">{{ 'errors.required' | translate }}</mat-error>
   </mat-form-field>
+
+  <div>
+    <mat-form-field class="example-full-width datepicker" appearance="fill">
+      <mat-label>{{ 'form-field-label.chose-deadline' | translate }}</mat-label>
+      <input autocomplete="none" matInput [matDatepicker]="picker" formControlName="taskDate">
+      <mat-datepicker-toggle matSuffix [for]="picker">
+        <mat-icon matDatepickerToggleIcon>keyboard_arrow_down</mat-icon>
+      </mat-datepicker-toggle>
+      <mat-datepicker #picker></mat-datepicker>
+      <mat-error *ngIf="addTaskForm.get('taskDate').hasError('required')">{{ 'errors.required' | translate }}</mat-error>
+    </mat-form-field>
+
```

Comment:
uselesss

Reply:
I can remove lines if it necessary, but I think code reads better with spaces.